src/adapter/src/catalog/builtin.rs

```diff
@@ -1973,6 +1973,13 @@ pub static MZ_DEFAULT_PRIVILEGES: Lazy<BuiltinTable> = Lazy::new(|| BuiltinTable
     is_retained_metrics_object: false,
 });
 
+pub static MZ_SYSTEM_PRIVILEGES: Lazy<BuiltinTable> = Lazy::new(|| BuiltinTable {
+    name: "mz_system_privileges",
+    schema: MZ_CATALOG_SCHEMA,
+    desc: RelationDesc::empty().with_column("privileges", ScalarType::MzAclItem.nullable(false)),
```

Comment:
Missing the id column? Test would have found that.

Reply:
I'm not sure that I follow, what id column? This table only has a single column of type `MzAclItem`.

Is that the issue? That we need some sort of primary key column?

I also don't think I understand the test comment either. There is a SLT test that queries this table and it succeeds.

Reply:
My mistake. I misread the document update!

Reply:
Er, I mean I misread how pack works. I saw it has `id` and `row` fields and thought you were missing the id field in the desc.